core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -971,8 +971,11 @@ private[group] class GroupCoordinator(
       if (validationErrorOpt.isDefined) {
         responseCallback(offsetMetadata.map { case (k, _) => k -> validationErrorOpt.get })
       } else {
-        groupManager.storeOffsets(group, memberId, offsetMetadata, responseCallback, producerId,
-          producerEpoch, requestLocal)
+        val offsets = offsetMetadata.map { case(k,v) => new TopicIdPartition(Uuid.ZERO_UUID, k) -> v }
```

Comment:
Please note that the signature of the offset commits for transactions are not changed, hence these conversions. It is possible to increase the scope to cover transaction commit offsets but erring on the side of caution here.

Reply:
I suppose that we can remove this now, isn't it?